lang/lua/stylua.talon

```diff
@@ -1,4 +1,4 @@
-tag: user.stylua
+code.language: stylua
```

Comment:
AFAICT stylua isn't a language but rather a linter/formatter. It shouldn't be turned into a language.